extensions/amp-access/0.1/amp-access.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assertHttpsUrl} from '../../../src/url';
+import {getService} from '../../../src/service';
+import {installStyles} from '../../../src/styles';
+import {isExperimentOn} from '../../../src/experiments';
+import {log} from '../../../src/log';
+
+
+/*eslint no-unused-vars: 0*/
```

Comment:
is this for the `let typdef` ? if so we can add a `Def` suffix and it should be ignored.